<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="displayOriginal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>350</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="displayOverlay">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>350</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="displayOverlayNOHE">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>50</y>
      <width>350</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="inputfile">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputfile_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for Overlay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputfile_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>830</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Histogram Equalized</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputfile_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>830</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputfile_5">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>830</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayHE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>350</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="displayBright">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>350</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="displayContrast">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>460</y>
      <width>350</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="brightnessSlider">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>810</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="contrastSlider">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>810</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="sliderPosition">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>810</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness Slider</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>810</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast Slider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputfile_6">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>410</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
